modules/mono/glue/GodotSharp/GodotSharp/Core/Extensions/NodeExtensions.cs

```diff
@@ -93,18 +93,22 @@ public partial class Node
         /// Negative indices access the children from the last one.
         /// To access a child node via its name, use <see cref="GetNode"/>.
         /// </summary>
-        /// <seealso cref="GetChildOrNull{T}(int)"/>
+        /// <seealso cref="GetChildOrNull{T}(int, bool)"/>
         /// <param name="idx">Child index.</param>
+        /// <param name="includeInternal">
+        /// If <see langword="true"/>, internal children are skipped (see <c>internal</c>
```

Comment:
Should be _"if `false`"_. I see it's a copy-paste so would need to be fixed in `Node.xml` as well.